src/gui/org/deidentifier/arx/gui/resources/Resources.java

```diff
@@ -286,6 +287,20 @@ public InputStream getStream(final String name) {
     
     /**
      * Returns an image.
+     * 
+     * @param type
+     * @return
+     */
+    public Image getImage(AttributeType type) {

```

Comment:
AttributeType is not an enum:

    /** Represents an identifying attribute. */
    public static AttributeType IDENTIFYING_ATTRIBUTE       = new AttributeType(ATTR_TYPE_ID);

    /** Represents a sensitive attribute. */
    public static AttributeType SENSITIVE_ATTRIBUTE         = new AttributeType(ATTR_TYPE_SE);

    /** Represents an insensitive attribute. */
    public static AttributeType INSENSITIVE_ATTRIBUTE       = new AttributeType(ATTR_TYPE_IS);

    /** Represents a quasi-identifying attribute. */
    public static AttributeType QUASI_IDENTIFYING_ATTRIBUTE = new AttributeType(ATTR_TYPE_QI);